src/main/kotlin/lotto/view/ResultView.kt

```diff
@@ -13,22 +14,33 @@ object ResultView {
     private const val SEPARATOR = "- "
     private const val TOTAL_RATE_PREFIX = "총 수익률은 "
     private const val TOTAL_RATE_SUFFIX = "입니다."
+    private const val ACCORDANCE_SUFFIX = "개 일치"
+    private const val BONUS_MESSAGE = ", 보너스 볼 일치"
 
     fun printLottos(lottoTicket: LottoTicket) {
         println("${lottoTicket.lottos.size}$BUY_COUNT_SUFFIX")
         printJoinToString(lottoTicket.lottos)
     }
 
-    fun printResult(result: LottoResult) {
+    fun printResult(result: LottoResult, money: Money) {
         println(WINNING_STATISTICS)
         println(WINNING_SEPARATOR)
-
         Rank.asList().forEach {
-            println("${it}$SEPARATOR${result.countByRank(it)}$COUNT_SUFFIX")
+            printRankResult(it, result)
         }
+        val totalRate = result.calculateTotalRate(money)
+        printTotalRate(totalRate)
+    }
+
+    private fun printRankResult(rank: Rank, result: LottoResult) {
+        val countMatch = String.format("%s$ACCORDANCE_SUFFIX", rank.countOfMatch)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val countMatch = "%s$ACCORDANCE_SUFFIX".format(rank.countOfMatch)
```